Portfolio/index.html

```diff
@@ -7,7 +7,34 @@
   </head>
```

Comment:
There should be only one level of heading hierarchy on a webpage, so consider using h4 instead of h3 here.